src/status_im2/common/qr_code/qr.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.common.qr-code.qr
```

Comment:
this component should actually be a quo2 component - https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=17928-209739&t=dgFUSXeZh0hk0GFV-0

Maybe you can move it to that namespace?
quo2.components.share.qr-code.view

We can do a follow up pr adding the preview screen etc then

Reply:
@J-Son89 : I had originally done that but then look at this conversation : 
https://github.com/status-im/status-mobile/pull/15528#discussion_r1154341154

I'll create that component in a separate PR.

Reply:
cool, thanks @siddarthkay .

Reply:
hmm, looking at this again. I'm not convinced that it should not be in quo2.
Sure the user-profile-qr-code should be in status-im2, but there should be a styled qr component in quo2 as there is many different use cases for this base component.

i.e quo2 should have a qr component as simple as :
```
(defn qr-code [{:keys [source width height]]
[rn/view
     {:style {:flex-direction  :row
              :justify-content :center}}
     [rn/image
      {:source source
       :style  {:width         width
                :height        height
                :border-radius 12}}]]
    ```
    We can then add the variants that use icons etc as shown in Figma. 
    
    Anyway this can all be done separate to this pr, so no need to block for now 👍

Reply:
yes @J-Son89 : that is the plan :)